### PR TITLE
add long and short name attributes for Red Hat build of Quarkus

### PR DESCRIPTION
**Pull request by inoxx03**
Hi, @oraNod 
I've added the attributes for the long and short version of  the product names for Red Hat build of Quarkus.
I added the `runtimes-` prefix to the attribute name to make them more easily distinguishable from similarly named product-specific attributes.
I have not added the `runtimes-` prefix to the attributes for other products, yet, but I can do so, if we decide to go with it.

Please let me know what you think.

### CODE REVIEW

**Comment by inoxx03** [comment]
@oraNod I've added a note to the attribute usage instructions, too. If you feel it might be too wordy, I can trim it down or rephrase it.

File: runtimes-common/attributes/runtimes-attributes.adoc [hunk 1]
```diff
@@ -10,13 +10,17 @@
 //
 
 // Each Runtimes product needs a unique name attribute for shared content.
+// When you write shared content, use the following attributes instead of the regular product name attributes of your project to ensure that correct product names are substituted when you reuse content across different documentation sets.
```
Comment (oraNod) [inline runtimes-common/attributes/runtimes-attributes.adoc]:
```suggestion
// When you write shared content, use the following attributes instead of product name attributes specific to your project. 
// These attributes ensure that correct product names are substituted in shared content sets. 
```

File: runtimes-common/attributes/runtimes-attributes.adoc [hunk 2]
```diff
@@ -10,13 +10,17 @@
...
+// When you write shared content, use the following attributes instead of the regular product name attributes of your project to ensure that correct product names are substituted when you reuse content across different documentation sets. 
 
 :rhdg-long: Red Hat Data Grid
 :rhdg-short: Data Grid
 
 :jws-long: Red Hat JBoss Web Server
 :jws-short: JBoss Web Server
 
+:runtimes-rhbq-long: Red Hat build of Quarkus
+:runtimes-rhbq-short: Quarkus
```
Comment (oraNod) [inline runtimes-common/attributes/runtimes-attributes.adoc]: @inoxx03 Yes, let's add the "runtimes-" prefix to these attributes to make them easier to distinguish with shared content.

I think it would also be easier for other writers if we avoided 4 letter abbreviations that are identifiable by project writers. An easy mistake might be typing "rhbq" instead of "rhdg". 

What do you think about ":runtimes-quarkus-long:" instead? Then maybe ":runtimes-datagrid-long:" etc.

**Review by oraNod — CHANGES_REQUESTED** [review]

File: runtimes-common/attributes/runtimes-attributes.adoc (hunk 2 above)
Comment (inoxx03) [reply runtimes-common/attributes/runtimes-attributes.adoc]: @oraNod Good point! Product acronyms are convenient, but might be unfamiliar to some writers.
Also, since we reuse the content across downstream docs only, there is IMHO little potential for confusion about whether the name refers to upstream or downstream.

I'll change the acronyms to more "friendly" names:)

**Comment by inoxx03** [comment]
@oraNod OK, I've updated the attribute names, please let me know if this works:)

**Review by oraNod — APPROVED** [review]

**Comment by oraNod** [comment]
thanks @inoxx03 great improvement!
